### PR TITLE
feat: project UI rework, move edit and delete buttons deeper

### PR DESCRIPTION
**Pull request by sjaanus**
1. Move edit and delete buttons deeper
2. Remove delete button from project list view
3. Add feature counter and text indication to delete button, so it is more clear when and why project can be deleted

image

### CODE REVIEW

File: frontend/src/component/project/ProjectAccess/ProjectAccess.styles.ts
Comment (sjaanus) [inline frontend/src/component/project/ProjectAccess/ProjectAccess.styles.ts:1]: This file was leftover from past.

File: frontend/src/utils/routePathHelpers.ts [hunk 1]
```diff
@@ -23,6 +25,11 @@ export const getCreateTogglePath = (
     return path;
 };
 
-export const getProjectEditPath = (projectId: string) => {
-    return `/projects/${projectId}/edit`;
+export const getProjectEditPath = (
+    projectId: string,
+    newProjectPath: boolean
+) => {
+    return newProjectPath
```
Comment (sjaanus) [inline frontend/src/utils/routePathHelpers.ts:32]: Edit will be on new path

File: frontend/src/component/common/FormTemplate/FormTemplate.styles.ts [hunk 2]
```diff
@@ -1 +1 @@
-export const formTemplateSidebarWidth = '27.5rem';
+export const formTemplateSidebarWidth = '36%';
```
Comment (sjaanus) [inline frontend/src/component/common/FormTemplate/FormTemplate.styles.ts:1]: Together with @nicolaesocaciu we came up with new width for API sidebar.

File: frontend/src/component/project/Project/ProjectSettings/Settings/DeleteProject.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    justifyContent: 'flex-end',
+    paddingTop: theme.spacing(3),
+}));
+
+interface IDeleteProjectProps {
+    projectId: string;
+    featureCount: number;
+}
+
+export const DeleteProject = ({
```
Comment (sjaanus) [inline frontend/src/component/project/Project/ProjectSettings/Settings/DeleteProject.tsx:32]: New component for project deletion

File: frontend/src/component/project/Project/ProjectSettings/Settings/EditProject.tsx [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+import useProjectForm, {
+    DEFAULT_PROJECT_STICKINESS,
+} from '../../hooks/useProjectForm';
+import { PageContent } from 'component/common/PageContent/PageContent';
+import { PageHeader } from 'component/common/PageHeader/PageHeader';
+import { DeleteProject } from './DeleteProject';
+import PermissionButton from 'component/common/PermissionButton/PermissionButton';
+import ProjectForm from '../../ProjectForm/ProjectForm';
+
+const EditProject = () => {
```
Comment (sjaanus) [inline frontend/src/component/project/Project/ProjectSettings/Settings/EditProject.tsx:25]: This is refactored component out of old EditProject, but in future old EditProject will be deleted

**Review by ivarconr — APPROVED** [review]

**Pull request merged**